### ISSUE TITLE
Add option to set path for python binary

### ISSUE THREAD

**Issue by Askaholic** [lexicalunit/atom-isort#25]
My binary is called `python3`, but it seems that this plugin keeps trying to use `python` which points to the python2 installation, because I keep getting an import error on `pathlib`.

**Comment by lexicalunit** [comment]
Shoot, I hadn't considered that scenario. I believe I can workaround this by first checking for `python3` and then falling back to `python` if not found.

Alternatively it could be a setting in the plugin where you set if you want it to use `python` or `python3` or whatever else ("python2" for example if you really wanted to). What do you think, @Askaholic?

**Comment by Askaholic** [comment]
I don't think it matters too much, but it's pretty typical for `python` to mean `python2` unless you're running in a virtual environment.

**Comment by lexicalunit** [comment]
Maybe historically but [python 2 has technically been sunset at this point](https://www.python.org/doc/sunset-python-2/) so I expect a lot of systems to assume `python` is the same as `python3` over time. Not that `atom-isort` shouldn't support it... I was more wondering if it'd be useful to you to be able to have it be configurable or if automatic detection would be fine.

**Comment by Askaholic** [comment]
For me the automatic detection would work fine. I have python2 installed only because it is installed by default (both on ubuntu and macos), but I never use it :P

**Comment by lexicalunit** [comment]
Alright, give v3.1.0 a try 👍

**Comment by Askaholic** [comment]
Hmm, now it's throwing an import error for `isort`. If I go to my terminal and run `python3` I can import `isort` though... Is there a way I can see what the python binary is that it's trying to call?

**Comment by lexicalunit** [comment]
First it tries to run a script using `python3` as the binary, then if it gets a "command not found" error from the shell it falls back to running the script using just `python` as the binary. As for where it looks for these binaries, it tries to find them at these locations: https://github.com/lexicalunit/atom-isort/blob/master/lib/index.js#L10 There's a bunch of likely paths for Windows and *NIX that it will search.

Note that you can also force it to use a specific python binary with the setting `atom-isort.pythonPath` if you'd like.

**Comment by Askaholic** [comment]
I just started getting another error:
```
Error: spawnSync python EPIPE
```
I'm on MacOS and python3 is installed in /usr/local/bin. I tried setting pythonPath both to /usr/local/bin/ and /usr/local/bin/python3 with no luck.

**Comment by lexicalunit** [comment]
I'll try and take a look at this later today, thank you for the debugging help!

**Comment by Askaholic** [comment]
Ha! Turns out I needed to restart atom. After updating it still had the old version of the plugin loaded so it was still trying to use python2. Now it seems to be working!

**Comment by lexicalunit** [comment]
Oh my gosh I forgot about this! Thanks for circling back, I was just so busy that I forgot to follow up and investigate this issue. I'm glad it seems to be working for you now though! 🎉

**Comment by Askaholic** [comment]
There is a weird visual glitch though where all of the text in the file flashes purple every time I save.

<img width="536" alt="Screen Shot 2020-07-17 at 11 10 25 AM" src="https://user-images.githubusercontent.com/17941539/87822619-4bf5bc00-c861-11ea-953e-b082f2b75caf.png">
